SU-SWS/saa_victoria_subtheme
# ADAPT-398  Hardcode Associates React App to a specific node

# READY FOR REVIEW

# Summary
- Hardcode Associates React App to a specific node

# Review By (Date)
- will update when PR is ready. Further discussion is needed regarding file structure/configuration

# Urgency
- low

# Steps to Test

1. Pull this branch to your local
2. Clear site cache
3. View `node/20`. You should be able to view the Associates React App

# Associated Issues and/or People
- [ADAPT-398](https://stanfordits.atlassian.net/browse/ADAPT-398)
- [ADAPT-399](https://stanfordits.atlassian.net/browse/ADAPT-399)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
